addons/vehicles/initSettings.sqf

```diff
@@ -0,0 +1,7 @@
+
+[QGVAR(keepEngineRunning), "CHECKBOX",
+[LSTRING(SettingKeepEngineRunningName), LSTRING(SettingKeepEngineRunningDesc)],
+localize ELSTRING(common,ACEKeybindCategoryVehicles),
+true, // default value
+true, // isGlobal
+{[QGVAR(keepEngineRunning), _this] call EFUNC(common,cbaSettings_settingChanged)}] call CBA_settings_fnc_init;
```

Comment:
Prettier formatting please. You took one of the old auto-generated files probably.